tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts

```diff
@@ -231,17 +231,30 @@ export class ExampleValue {
         return instance;
     }
 
-    private static createSchemaForDfeObject(raw: any, dfeFormat: string): ObjectSchema | undefined {
+    private static createSchemaForDfeObject(session: Session<TestCodeModel>, raw: any, dfeFormat: string): ObjectSchema | undefined {
```

Comment:
Will this breaking change break other scenarios?

Reply:
no, this method is added in my previous check-in and not used in other scenarios.